.github/workflows/main.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - const-generics-tosql
```

Comment:
Please, revert this change.

Reply:
Of course. Am I missing a trick to get Actions to run on my branch without that?